applications/nrf5340_audio/README.rst

```diff
@@ -309,299 +309,13 @@ Software codec requirements
-|TP62         | P1.06                      | UART1 CTS                                        | 1.0mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP63         | P1.07                      | UART1 RTS                                        | 1.0mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP64         | NetJ5-10                   | SD card slot card detect                         | 1.0mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP65         | P0.11                      | SD card slot level translator enable             | 1.0mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP66         | P1.15                      | Current shunt monitor alert signal               | 1.0mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP67         | GND                        | Ground                                           | 1.5mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP68         | LINE_IN.LEFT               | Line-in jack tip                                 | 1.5mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-|TP69         | LINE_IN.RIGHT              | Line-in jack ring                                | 1.5mm | Top    |
-+-------------+----------------------------+--------------------------------------------------+-------+--------+
-
-
-.. _nrf53_audio_hw_limitations:
-
-nRF5340 Audio hardware limitations
-----------------------------------
-
-The following table lists hardware limitations discovered in different revisions of the nRF5340 Audio DK.
-
-.. list-table::
-    :widths: auto
-    :header-rows: 1
-
-    * - PCA10121 revision
-      - Limitation
-      - Description
-      - Workaround
-      - Fixed in revision
-    * - Rev 1.0.0
-      - CS47L63 AD-DA converter (**U2**) may fail to start
-      - In some occasions, the 1.2 V power supply for **U2** is not provided at boot-up.
-        This is caused by higher than expected inrush current.
-        This function is tested in production.
-        The issue should not happen, although we observe that some kits have the problem.
-      - Restart kit or attach the battery to the kit before connecting the USB cable.
-        If problem persists, contact Nordic Semiconductor and ask for replacement.
-      - Rev 1.0.1
+Read the `nRF5340 Audio DK Hardware`_ documentation on Nordic Semiconductor Infocenter for more information about this development kit.
```

Comment:
Maybe?:

```suggestion
Read the `nRF5340 Audio DK`_ hardware documentation on Nordic Semiconductor Infocenter for more information about this development kit.
```

Reply:
Nope, that's the title of the section on Infocenter... :(